### PR TITLE
LPD-23945 LRCI-4187 Add playwright test batch in release tester

### PR DESCRIPTION
**Pull request by john-co**
**JIRA Ticket**
https://liferay.atlassian.net/browse/LPD-23945
https://liferay.atlassian.net/browse/LRCI-4187

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:playwright

**Comment by john-co** [comment]
[Testray Comparison](https://testray.liferay.com/home/-/testray/runs/compare?formDate=1714195055765&testrayRunIdA=2564087152&testrayRunIdB=2561315627) from earlier testing in https://github.com/john-co/liferay-portal/pull/542

**Comment by john-co** [comment]
[Playwright test total](https://testray.liferay.com/home/-/testray/case_types/metrics?tab=results&testrayBuildId=2564087135) from release tester matches and has similar number of failures compared with [Acceptance commit near U118](https://testray.liferay.com/home/-/testray/case_types/metrics?tab=results&testrayBuildId=2560115516).

File: test.properties [hunk 1]
```diff
@@ -7575,7 +7578,11 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         functional-bundle-ubuntu18-tomcat90-mysql57-jdk8,\
         functional-smoke-bundle-wildfly180-mysql57-jdk8,\
         functional-smoke-bundle-wildfly230-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly261-mysql57-jdk8
+        functional-smoke-bundle-wildfly261-mysql57-jdk8,\
+        playwright-js-bundle-tomcat90-mysql57-jdk8
+
+    test.batch.names[portal-release-playwright]=\
```
Comment (brittneyq) [inline test.properties:7584]: hi @john-co ! Your changes LGTM based on the results for release-tester and the playwright test suite. Just for SF purposes, could you move test.batch.names[portal-release-playwright]=\ into its own section and sort it with the other portal release suites? Similar to https://github.com/liferay/liferay-portal/blob/master/test.properties#L8309-L8316

File: test.properties [hunk 2]
```diff
@@ -952,6 +952,9 @@
         style-book-web,\
         users-admin-web
 
+    playwright.test.project[playwright-js-bundle-tomcat90-mysql57-jdk8]=\
```
Comment (brittneyq) [inline test.properties:955]: Oh and can you resort this too? Thanks!

**Comment by john-co** [comment]
Thanks @brittneyq! I made the sort adjustment. Let me know if there's any other feedback before I forward.

**Comment by john-co** [comment]
Pushed to https://github.com/pyoo47/liferay-portal/pull/3195

**Pull request closed without merging**